airflow/providers/google/cloud/operators/dataflow.py

```diff
@@ -377,6 +394,7 @@ def __init__(
         self.poll_sleep = poll_sleep
         self.job_id = None
         self.hook: Optional[DataflowHook] = None
+        self.options.update(dataflow_default_options)
```

Comment:
Would not this override user provided options with default options?

Reply:
Good point. I will fix it.